data_structures/src/chain.rs

```diff
@@ -93,12 +97,22 @@ impl fmt::Display for Environment {
 }
 
 impl Environment {
+    /// Returns the Bech32 prefix used in this environment: "wit" in mainnet and "twit" in testnet
     pub fn bech32_prefix(&self) -> &str {
         match self {
             Environment::Mainnet => "wit",
             _ => "twit",
         }
     }
+
+    /// Returns true if the consensus constants can be modified in this environment.
+    /// This is only allowed in a development environment.
+    pub fn can_modify_consensus_constants(&self) -> bool {
```

Comment:
Maybe `override` is more explicit about the fact that there are some defaults that you can optionally customize?
```suggestion
    pub fn can_override_consensus_constants(&self) -> bool {
```